### PR TITLE
[Graph] Add Flip node.

### PR DESCRIPTION
**Pull request by mciprian13**
**Summary**
- Add Flip node inspired from Numpy.
- The node reverts the order of the elements along a given axis.
- The node currently supports single axis: multiple nodes can be chained for multiple axes.
- The node can be used for example for RGB <-> BGR conversion inside the graph.

**Documentation**
None

**Test Plan**
Unit tests for operator, ONNX importer and exporter.

### CODE REVIEW

File: tests/unittests/OperatorTest.cpp [hunk 1]
```diff
@@ -3079,6 +3079,302 @@ TEST_P(OperatorTest, TransposeIntoReshapeOptim) {
...
+static void testFlip(glow::PlaceholderBindings &bindings, glow::Module &mod,
+                     glow::Function *F, glow::ExecutionEngine &EE,
+                     std::vector<elemType> inputData,
+                     std::vector<elemType> expectedData,
+                     llvm::ArrayRef<dim_t> dims, dim_t axis,
+                     ElemKind elemKind = ElemKind::FloatTy) {
+
+  // Create network.
+  Placeholder *input;
+  if (isQuantizedElemKind(elemKind)) {
```
Comment (jfix71) [inline tests/unittests/OperatorTest.cpp]: nit: We have a helper `createPlaceholderConditionallyQuantized()` which I believe does this exact thing.

File: tools/ClassGen/InstrGen.cpp [hunk 2]
```diff
@@ -679,6 +679,18 @@ int main(int argc, char **argv) {
       .autoVerify(VerifyKind::SameElementType, {"Dest", "Src"})
       .autoIRGen();
 
+  //===--------------------------------------------------------------------===//
+  //                Reorder transformations
+  //===--------------------------------------------------------------------===//
+
+  BB.newInstr("Flip")
```
[the remark this answers is not in the document]
Comment (mciprian13) [reply tools/ClassGen/InstrGen.cpp]: The implementations for both the Interpreter and the CPU do NOT allow for in-place computation (for in-place computation the implementation would be trickier).

File: tests/unittests/OperatorTest.cpp (hunk 1 above)
Comment (mciprian13) [reply tests/unittests/OperatorTest.cpp]: Done.

**Comment by mciprian13** [comment]
Can we have this landed? Thanks!

**Comment by mciprian13** [comment]
@jfix71 If there is nothing left to be modified, I think it is ready to be landed.

File: tools/ClassGen/InstrGen.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (jfix71) [reply tools/ClassGen/InstrGen.cpp]: Right, I was suggesting we could change the implementations to be in place to reduce an unnecessary buffer, but we can land it as is/consider it for the future if desired.

**Pull request closed without merging**